core/mobile/src/backend/MobileHost.ts

```diff
@@ -195,8 +193,10 @@ export class MobileHost {
       // following will provide impl for device specific api.
       setupMobileRpc();
     }
+    const iModelHostConfiguration = opt?.iModelHost ?? new IModelHostConfiguration();
+    iModelHostConfiguration.authorizationClient = authorizationClient;
```

Comment:
Is this a fix for the backend auth failure Todd was seeing?

Reply:
Yes.